tempto-core/src/main/java/io/prestodb/tempto/internal/convention/tabledefinitions/ConventionTableDefinitionDescriptor.java

```diff
@@ -95,6 +106,16 @@ public ConventionTableDefinitionDescriptor(Path ddlFile)
         }
     }
 
+    private long copy(Path dataFile, Process process)
```

Comment:
`s/copy/feed/`

Reply:
hm... it is just a wrapper of `Files.copy`. It just suppresses the exception.

Reply:
But is very specific usage: to feed input to a process. You rather don't call it "copying".